### PR TITLE
Added Second Page + Charts

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

- Adds second page + charts to bill generation! 
- Adds ability to regenerate PDF independently from regenerating bill, making generator much more customizable
- incorporated grayson's feedback

### How to Review

sanity check + trust me? :P We'll need to test with Grayson one-on-one later

### Next steps

- Need to incorporate grayson's email about would be costs and finalize bill cost calculation

### Tests Performed, Edge Cases

I've tested pretty thoroughly with just generating bills. but this is one of those things that needs more real data to truly test

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: utils/charts/lineChart.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    series: [
+      {
+        name: "What You've Paid",
+        data: trueCosts,
+        color: '#cd6795'
+      },
+      {
+        name: '"Would-be" costs',
+        data: wouldBeCosts,
+        color: '#747474'
```
Comment (dfangshuo) [inline utils/charts/lineChart.js:26]: maybe declare the colors as constants at the top of the file, or together in a separate colors constants file?

File: utils/pdfgeneration.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+// Generate charts for both subscriber bills based on bill data
+
+const generateChartsForSubscriberBill = async (
+  subscriber,
+  latestBill,
+  previousBills
+) => {
+  console.log(`Generating Generation Data Bar Chart for ${subscriber.name}`);
+  const generationChart = generateGenerationDataChart(latestBill);
+  await saveChartToFile(generationChart, `${latestBill.id}_chart1`);
```
Comment (dfangshuo) [inline utils/pdfgeneration.js:57]: nit

```suggestion
  const chartName = `${latestBill.id}_chart1`;
  await saveChartToFile(generationChart, chartName);
```

Comment (dfangshuo) [reply utils/pdfgeneration.js:57]: same suggestion for below as well

File: utils/pdfgeneration.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  // Create a unique timestamp to ensure unique URL for pdf
+  const pdfStamp = new Date().getTime();
+  await ReactPDF.render(
+    <BillTemplate
+      subscriber={subscriber}
+      solarProject={solarProject}
+      subscriberBill={latestBill}
+      previousBills={previousBills}
+    />,
+    `./temp/${latestBill.id}_${pdfStamp}.pdf`
```
Comment (dfangshuo) [inline utils/pdfgeneration.js:97]: i would make the 'temp' folder name a variable on Constants.js just so it's easier to modify/isn't hardcoded

Comment (dfangshuo) [reply utils/pdfgeneration.js:97]: it's used in quite a lot of places (below), and I foresee a lot of pain needing to change them in all of those places

Comment (aivantg) [reply utils/pdfgeneration.js:97]: eh, in what world would the name of the temp folder ever be changed tho?? It serves a pretty standard purpose, does not seem worth

Comment (dfangshuo) [reply utils/pdfgeneration.js:97]: as it is right to have such a folder hardcoded in different places creates a loosely coupled arrangement that is prone to errors/bugs in the future. Not only does declaring this in a constant make this easier to change, but it forestalls errors like that
